tests/task.test.ts

```diff
@@ -12,135 +12,223 @@ const index = {
-      expect(stausReponse).toHaveProperty('finishedAt', expect.any(String))
-      expect(stausReponse).toHaveProperty('startedAt', expect.any(String))
+      const task = await client.getTask(enqueuedTask.taskUid)
+
+      expect(task.indexUid).toEqual(index.uid)
+      expect(task.status).toEqual(TaskStatus.TASK_SUCCEEDED)
+      expect(task.type).toEqual(TaskTypes.DOCUMENTS_ADDITION_OR_UPDATE)
+      expect(task.enqueuedAt).toBeDefined()
+      expect(task.uid).toEqual(enqueuedTask.taskUid)
+      expect(task).toHaveProperty('details')
+      expect(task.details.indexedDocuments).toEqual(7)
+      expect(task.details.receivedDocuments).toEqual(7)
+      expect(task.duration).toBeDefined()
+      expect(task.enqueuedAt).toBeDefined()
+      expect(task.finishedAt).toBeDefined()
+      expect(task.startedAt).toBeDefined()
     })
 
-    test(`${permission} key: Get all updates`, async () => {
+    test(`${permission} key: Get one task with index instance`, async () => {
       const client = await getClient(permission)
-      const { uid } = await client.index(index.uid).addDocuments([{ id: 1 }])
-      await client.waitForTask(uid)
+      const enqueuedTask = await client.index(index.uid).addDocuments(dataset)
+      await client.waitForTask(enqueuedTask.taskUid)
+
+      const task = await client.index(index.uid).getTask(enqueuedTask.taskUid)
+
+      expect(task.indexUid).toEqual(index.uid)
+      expect(task.status).toEqual(TaskStatus.TASK_SUCCEEDED)
+      expect(task.type).toEqual(TaskTypes.DOCUMENTS_ADDITION_OR_UPDATE)
+      expect(task.enqueuedAt).toBeDefined()
+      expect(task.uid).toEqual(enqueuedTask.taskUid)
+      expect(task).toHaveProperty('details')
+      expect(task.details.indexedDocuments).toEqual(7)
+      expect(task.details.receivedDocuments).toEqual(7)
+      expect(task.duration).toBeDefined()
+      expect(task.enqueuedAt).toBeDefined()
+      expect(task.finishedAt).toBeDefined()
+      expect(task.startedAt).toBeDefined()
+    })
+
+    test(`${permission} key: Get all tasks`, async () => {
+      const client = await getClient(permission)
+      const enqueuedTask = await client
+        .index(index.uid)
+        .addDocuments([{ id: 1 }])
+      await client.waitForTask(enqueuedTask.taskUid)
 
-      const response = await client.index(index.uid).getTasks()
+      const tasks = await client.getTasks()
 
-      expect(response.results[0]).toHaveProperty(
+      expect(tasks.results[0]).toHaveProperty(
         'status',
         TaskStatus.TASK_SUCCEEDED
       )
-      expect(response.results[0]).toHaveProperty('uid', expect.any(Number))
-      expect(response.results[0].type).toEqual('documentAddition')
-      expect(response.results[0]).toHaveProperty('duration', expect.any(String))
-      expect(response.results[0]).toHaveProperty(
-        'enqueuedAt',
-        expect.any(String)
+      expect(tasks.results[0].indexUid).toEqual(index.uid)
```

Comment:
Do you really need to test all the properties here? Because the test above, covered the same spots, correct?